pennylane/devices/default_clifford.py

```diff
@@ -210,6 +254,53 @@ def circuit():
         Maintaining and working with this tableau representation instead of the complete state vector
         makes the calculations of increasingly large Clifford circuits more efficient on this device.
 
+    .. details::
+        :title: Probabilities for Basis States
+        :href: clifford-probabilities
+
+        As the ``default.clifford`` device supports executing quantum circuits with a large number of qubits,
+        we restrict the ability to compute the ``analytical`` probabilities for `all` computational basis
+        states at once to maintain computational efficiency when system size scales beyond ``24`` qubits.
+        Instead, one can sepcify the target basis state(s), i.e., subset `basis state(s)` of interest,
+        either during the initialization of device via the ``target_states`` keyword argument,
+        or at any other stage before executing the circuit by setting the ``probability_target``
+        property of the device.
+
+        .. code-block:: python
+
+            import pennylane as qml
+            import numpy as np
+
+            basis_states = np.array([[0, 0], [1, 0]])
+            dev = qml.device("default.clifford", target_states = basis_states)
+
+        After doing this, one can simply use the :func:`qml.probs <pennylane.probs>` with its usual arguments
+        and probabilities for the specified target states would be computed and returned.
```

Comment:
Could also expand just a little bit what those usual arguments are, i.e. what is up with those operators?